hack/hive-generate-config.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+HIVE_REPO="https://github.com/openshift/hive.git"
```

Comment:
This would be a good use of Bash's parameter expansion so these variables can be set on the local execution environment, with the current values as a default if they are not set:
```bash
HIVE_REPO="${HIVE_REPO:=https://github.com/openshift/hive.git}"
```